notifi-network/notifi-sdk-ts
# MVP-1472 [1/x] Create Alert Notification Panel (for Broadcast Messages only)

Alert Notifications for Broadcast Messages.

To Test:
- Create a configuration in the config tool
- Go to your test app with the React card implementation. Make sure that you have subscribed to that card.
- Send a broadcast message via the config tool by
1. Create an Broadcast Event type with an id.
2. Go the Send Messages page.
3. In the broadcast id, enter in the [dappname]_[idname]. If you are using the junitest.xyz login to create your config, it will be junitest.xyz_whateveryouusedastheidvalueinthebroadcastevent.
- Send broadcast messages with a variety of short and long subjects / messages to make sure that they render properly in the react card
- Now go to the React Card in your test app. Disconnect and reconnect. 
- Make sure that it goes to the Alert History page first and you see the broadcast messages listed. We limit the "preview" of messages to one line for the subject and message each. 
- Click on Manage Alerts -> Send you to the Preview Card.
- Back Arrow will send you back to Alert History.

Dark Mode:

https://user-images.githubusercontent.com/108922885/197430531-e2ab4ca5-9bb9-4793-a93b-5c80ff92e60e.mov

Light Mode: 

https://user-images.githubusercontent.com/108922885/197430278-8c98e374-5dd3-4ff4-8341-adf2c7cca438.mov

Narrow Card:

<img width="499" alt="Screen Shot 2022-10-23 at 6 32 25 PM" src="https://user-images.githubusercontent.com/108922885/197430582-06429e21-679d-4815-aba8-c35ed77dacb0.png">
